mmv1/templates/terraform/examples/redis_cluster_ha.tf.erb

```diff
@@ -1,5 +1,4 @@
 resource "google_redis_cluster" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
this example needs to be marked as GA by removing the line: https://github.com/GoogleCloudPlatform/magic-modules/pull/9205/files#diff-a2190d3f07fbb268a97573b4ac3e7705e0da8c09678a6417ecbf9393f662319bR54

Reply:
Done